promptops/promptops/skill_export.py

```
@@ -57,42 +63,127 @@ def generate_skill_content(data: Dict[str, Any], raw_data: Dict[str, Any], raw_c
+    if not health_report:
+        lines.append("🎉 **All skills are healthy!**")
+    else:
+        for report in health_report:
+            lines.append(f"## `{report['file']}`")
+            for err in report["errors"]:
+                lines.append(f"- **{err['type']}**: {err['message']} (Line: {err.get('line', 'N/A')})")
+            lines.append("")
+            
+    dashboard_path.write_text("\n".join(lines), encoding='utf-8')
+
 def process_skills(prompts_path: Path, docs_path: Path):
```

Ruff (0.15.15)

[warning] Too many branches (19 > 12)

(PLR0912)